### ISSUE TITLE
Improve thermostatic controls available with AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl object

### ISSUE THREAD

**Issue by chrisbalbach** [NREL/EnergyPlus#5386]
The AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl object appears to offer far more limited thermostatic control options when compared to the options available for the AirConditioner:VariableRefrigerantFlow objects. 

The multiple thermostatic control options available for the AirConditioner:VariableRefrigerantFlow object represent more realistic control options available in real world VRF equipment.

**Comment by kbenne** [comment]
@rongpengzhang I believe you are the correct person to receive this, but if I am wrong then please pass it off to the correct person. I am under the understanding that you are currently working in this area.

**Comment by rongpengzhang** [comment]
@chrisbalbach @kbenne Yes, the current AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl object only offers limited thermostatic control options, and only support HP operations. We have been working on the physics based VRF-HR model, in which the thermostatic control options will also be expanded.
